joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/Main.scala

```diff
@@ -6,8 +6,11 @@ import scopt.OParser
 
 /** Command line configuration parameters
   */
-final case class Config(inputPath: String = "", outputPath: String = X2CpgConfig.defaultOutputPath)
-    extends X2CpgConfig[Config] {
+final case class Config(
+  inputPath: String = "",
+  outputPath: String = X2CpgConfig.defaultOutputPath,
+  android: String = null
```

Comment:
If possible, could you try to use Scala's `Option` classes? e.g. `android: Option[String] = None` and then just wrap incoming values by `Option(value)`. Will keep things much more null-safe, and in Joern at least we avoid passing around `null` values.

Reply:
Sure! That is much better.